man/systemd.journal-fields.xml

```diff
@@ -272,21 +272,11 @@
 
       <varlistentry>
         <term><varname>_SOURCE_REALTIME_TIMESTAMP=</varname></term>
-        <term><varname>_SOURCE_MONOTONIC_TIMESTAMP=</varname></term>
```

Comment:
hmm? Why drop this?

Reply:
As we cannot fix the timestamp, the field should not be used by any user application, hence I do not think we should document it.